src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -107,6 +112,7 @@ export const ProfilePage = () => {
               <Column width="full">
                 <AnimatedBox
                   flexShrink="0"
+                  position={isMobile && (isFiltersExpanded || isBagExpanded) ? 'fixed' : 'static'}
```

Comment:
non-blocking: this does work but because of the check `!isMobile || !isFiltersExpanded` above I think you can actually just do `isMoblie && isBagExpanded` because it will never be `isMobile && isFiltersExpanded`